### DISCUSSION TITLE
How to create a correlation matrix in Orange

### DISCUSSION THREAD

**Discussion by rehoyt** [biolab/orange3#5926] [General]
I thought when I saw the heatmap widget that it would create a correlation matrix of continuous variables. Instead, it appears to help visualize hierarchical clustering. Orange includes a correlation widget, but I don't see a way to generate a correlation matrix that would be helpful for teaching, etc. Below is a typical example
![image](https://user-images.githubusercontent.com/25651922/160937874-5620507e-d1e6-4643-8486-cfdb50956046.png)

**Comment by TonciG** [comment]
Have asked my self the same question but no success. i do not think this is possible with the current (3.32) version of Orange.
